install/k8s/safe/Deployment.yaml

```diff
@@ -63,8 +63,12 @@ spec:
           value: "spiffe://aegis.ist/workload/aegis-safe/ns/aegis-system/sa/aegis-safe/n/"
         - name: AEGIS_SAFE_DATA_PATH
           value: "/data"
+        - name:  AEGIS_SAFE_AGE_KEY_SECRET_NAME
+          value: "aegis-safe-age-key"
```

Comment:
this is new. 

if you want, you can use a different secret name to store the aegis master key.